src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -318,7 +328,31 @@ export class SkyFlyoutComponent implements OnDestroy, OnInit {
     }
   }
 
-  public onMouseDown(event: MouseEvent): void {
+  public onHeaderGrabHandleMouseDown(event: MouseEvent): void {
+    this.onResizeHandleMouseDown(event);
+  }
+
+  public onHeaderGrabHandleKeyDown(event: KeyboardEvent): void {
+    switch (event.key.toLowerCase()) {
```

Comment:
We probably don't need to run `toLowerCase` on the key name anymore since the capitalization is consistent on all browsers (including IE 11). Also, this will avoid the errors we've been seeing, RE "Cannot call toLowerCase of undefined".

Reply:
Good point. I also added null check - thanks for reminding me about that error.